pkg/translator/prometheus/normalize_name.go

```diff
@@ -131,7 +131,7 @@ func normalizeName(metric pmetric.Metric, namespace string) string {
 	// Split metric name in "tokens" (remove all non-alphanumeric)
 	nameTokens := strings.FieldsFunc(
 		metric.Name(),
-		func(r rune) bool { return !unicode.IsLetter(r) && !unicode.IsDigit(r) },
+		func(r rune) bool { return !unicode.IsLetter(r) && !unicode.IsDigit(r) && string(r) != "_" },
```

Comment:
Please do not update *normalizeName()*. This function is called only when feature gate for normalization is ON. In this case, metric names that are produced must conform to Prometheus convention (i.e. no double underscores).